modules/home-manager/sops.nix

```
@@ -135,7 +135,7 @@ in {
 
     defaultSymlinkPath = lib.mkOption {
       type = lib.types.str;
-      default = "%r/secrets";
+      default = "${config.home.homeDirectory}/.secrets";
```

After some thought it might be best to stick to xdg conventions:

```suggestion
      default = "${config.xdg.configHome}/sops-nix/secrets";
```